client/src/api/v3/boost.ts

```diff
@@ -0,0 +1,30 @@
+import { emit } from 'api/common/socket';
+import { BoostConfigType } from 'types/boost';
+
+/**
+ * Send the content of the given configuration file on `boost/configs/action` over MQTT
+ *
+ * @param type the type of the configuration being sent
+ * @param configFiles list of files, only the first would be considered
+ */
+export default function uploadConfig(
+  type: BoostConfigType,
+  configFiles: FileList,
```

Comment:
It would be neater for this to take a `File` rather than a `FileList`, since we only ever use the first file. That way, we don't have to worry about the list being empty, the list containing lots of items somehow, etc.

Reply:
Yeah I had that in mind but completely forgot about it, thanks for picking that up!